packages/riipen-ui/src/components/Input.jsx

```diff
@@ -133,6 +134,18 @@ const Input = ({
         .warning {
           border-color: ${theme.palette.warning.main};
         }
+
+        .small {
+          padding: ${theme.spacing(1)}px;
+        }
+
+        .medium {
+          padding: ${theme.spacing(2)}px;
+        }
+
+        .large {
+          padding: ${theme.spacing(4)}px;
+        }
```

Comment:
hmmm i wonder if we want to scale the text size instead of adding padding? The places i think we would use it would be something like the course wizard name input?

Reply:
Yeah I was wondering that too.  Any suggestions on font size? I'd assume `small` would be something like body1, but not sure how big (and bold?) large should get.
I'll give it a try and report back with screenshots!